module/device/method/wsa.py

```diff
@@ -9,6 +9,10 @@
 from module.exception import RequestHumanTakeover
 from module.logger import logger
 
+import win32gui
+import ctypes
+import win32con
+import win32api
```

Comment:
Import these at the beginning of `display_resize_wsa()`, or linux will import the dependencies from windows